pallets/gear-eth-bridge/src/lib.rs

```diff
@@ -374,6 +397,16 @@ pub mod pallet {
             // Returning appropriate type.
             Some(proof.into())
         }
+
+        /// The account ID of the bridge admin.
+        pub fn bridge_admin_account_id() -> T::AccountId {
```

Comment:
pls add tests for accounts remain unchanged: since they depend on pallet id which is potentially subject to change

Reply:
basically, I'd maybe put them as config constants for external clients as js ease of query

Reply:
Added